### PR TITLE
dep: updated package dependancies

### PR DESCRIPTION
**Pull request by southbite**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -202,17 +201,29 @@ MongoProvider.prototype.upsert = function(path, setData, options, dataWasMerged,
...
 
   _this.db.findAndModify({'path': path}, setParameters, function (err, response) {
 
-    if (err) return callback(err);
+    if (err) {
 
-    callback(null, response, response, true, _this.__getMeta(response));
+      if (err.toString().indexOf('duplicate key error collection') > -1) {
+
+        //1 retry - as mongo doesn't seem to understand upsert:true on a unique index means...
```
Comment (JohanvdWest) [inline index.js]: This seems strange. Why would it pass the second time?

File: lib/datastore.js [hunk 2]
```diff
@@ -188,15 +188,18 @@ MongoDataStore.prototype.ObjectID =  mongodb.ObjectID;
...
 
   var _this = this;
-  mongoclient.connect (_this.config.url, _this.config.opts, function (err, database) {
+
+  mongoclient.connect (_this.config.url, _this.config.opts, function (err, client) {
 
     if (err) return callback(err);
 
-    var collection = database.collection(_this.config.collection);
+    var db = client.db(_this.config.database);
```
Comment (JohanvdWest) [inline lib/datastore.js]: Don't see this being used.

Comment (southbite) [reply lib/datastore.js]: it is used in the following line: var collection = db.collection(_this.config.collection);

Comment (southbite) [reply lib/datastore.js]: collection and client are then used going forward

Comment (JohanvdWest) [reply lib/datastore.js]: Sorry, missed that.

**Pull request merged**